Assets/Scripts/Enemies/Bosses/SpikeBall.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UnityEngine;
+
+public class SpikeBall : EnemyMovement
+{
+
+    float timer = 0;
+
+    public override Vector3 PositionTransform(EnemyPosition position, float deltaTime)
+    {
+        return position.Up * deltaTime;
+    }
+
+    public override Vector3 RotationTransform(EnemyPosition position, float deltaTime)
+    {
+        return Vector3.zero;
+    }
+
+    public override void Update()
+    {
+        base.Update();
```

Comment:
It would be cleaner but then it would provide enemies that wouldn't use the method the method.  This way it is made for enemies who will actually use it.